src/main/java/com/app/backend/controller/DecedentController.java

```diff
@@ -0,0 +1,52 @@
+package com.app.backend.controller;
+
+import com.app.backend.model.Decedent;
+import com.app.backend.repository.DecedentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+
+@RestController
+@RequestMapping("/api/decedent")
+@CrossOrigin("*")
+public class DecedentController {
+
+    @Autowired
```

Comment:
Autowired is unneccesery here. DataSource for example is better. Autowired can be pretty hectic and hard to control